src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
I'm currently limited by the only version is ALLOWED(**otherwise I get a build failure**) in the project.json file is whatever specified in:
<ExpectedPrerelease>rc3-23729</ExpectedPrerelease>.
 
@dagood  unfortunately it's no longer work as for the reason above.

@roncain the design you proposed will need corefx level build scripts fixes. Also, allow testing any version is an edge scenario. I think we should unblock ourselves for the time being.